### PR TITLE
added custom headers into .ok response

### PR DESCRIPTION
**Pull request by yuri-qualtie**
PR is implementation proposal for #496

### CODE REVIEW

File: Xcode/Sources/HttpResponse.swift [hunk 1]
```diff
@@ -158,7 +161,7 @@ public enum HttpResponse {
 
     func content() -> (length: Int, write: ((HttpResponseBodyWriter) throws -> Void)?) {
         switch self {
-        case .ok(let body)             : return body.content()
+        case .ok(let body, _)             : return body.content()
```
Comment (michaelenger) [inline Xcode/Sources/HttpResponse.swift]: Please remove spaces before the `:` to keep it in line with the lines below.

**Comment by michaelenger** [comment]
I like this change, but is it an idea to make custom headers available to the other response types (not found, for example) as well?

File: Xcode/Sources/HttpResponse.swift (hunk 1 above)
Comment (yuri-qualtie) [reply Xcode/Sources/HttpResponse.swift]: Done

**Comment by yuri-qualtie** [comment]
> I like this change, but is it an idea to make custom headers available to the other response types (not found, for example) as well?

Maybe it could be done later. To my mind, currently custom headers are useful for .ok responses

**Comment by michaelenger** [comment]
I forgot to mention that you should update the CHANGELOG with your change. You can find it at [CHANGELOG.md](https://github.com/httpswift/swifter/blob/stable/CHANGELOG.md).

**Comment by yuri-qualtie** [comment]
> I forgot to mention that you should update the CHANGELOG with your change. You can find it at [CHANGELOG.md](https://github.com/httpswift/swifter/blob/stable/CHANGELOG.md).

Done @michaelenger

**Comment by gabrieldonadel** [comment]
@Vkt0r, any change of release a new version of Swifter including this change?

**Pull request merged**